pkgs/tools/security/cfssl/default.nix

```diff
@@ -1,29 +1,55 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, fetchpatch }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+let
+  # Embed static files in the built-in webserver
+  rice = buildGoModule rec {
+    name = "rice";
+    src = fetchFromGitHub {
+      owner = "GeertJohan";
+      repo = "go.rice";
+      rev = "v1.0.0";
+      sha256 = "0m1pkqnx9glf3mlx5jdaby9yxccbl02jpjgpi4m7x1hb4s2gn6vx";
+    };
+    vendorSha256 = "0cb5phyl2zm1xnkhvisv0lzgknsi93yzmpayg30w7jc6z4icwnw7";
+    subPackages = [ "rice" ];
+  };
```

Comment:
This should probably be moved to its own derivation.

Reply:
I assume, this is considered ok then within this PR's scope. That policy doubt was was keeping ke back.

Reply:
@kalbasit Done. Astonishingly it still works on first attempt :smile: . Thanks for the suggestion!